tests/foreman/cli/test_satellitesync.py

```diff
@@ -64,13 +64,58 @@ def config_export_import_settings():
     Settings.set({'name': 'content_disconnected', 'value': disconnected_mode_value})
 
 
+@pytest.fixture(scope='function')
+def export_cleanup_function(default_sat, function_org):
+    """Deletes export/import dirs of function org"""
+    yield
+    # Deletes directories created for export/import test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}').stdout == ''
+    assert default_sat.execute(f'rm -rf {IMPORT_DIR}/{function_org.name}').stdout == ''
+
+
+@pytest.fixture(scope='function')
+def export_cleanup_module(default_sat, module_org):
+    """Deletes export/import dirs of module_org"""
+    yield
+    # Deletes directories created for export/import test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{module_org.name}').stdout == ''
+    assert default_sat.execute(f'rm -rf {IMPORT_DIR}/{module_org.name}').stdout == ''
+
+
+def validate_filepath(sat_obj, org):
+    """Checks the existence of certain files in a dir"""
+    result = sat_obj.execute(
+        fr'find {EXPORT_DIR}{org.name} -type f \( -name "*.json" -o -name "*.tar.gz" \)'
+    )
+    return result.stdout
+
+
+def move_pulp_archive(sat_obj, org, export_message):
```

Comment:
Though it may only be used in this test module, this is a function that I'd suggest exist on the `robottelo.hosts.Capsule` class, since its first argument is a satellite object.